hoomd/operations.py

```diff
@@ -1,47 +1,112 @@
+    in HOOMD-blue,  `Operations` objects manage multiple independent
+    sequences described below.
+
+    The types of operations which can be added to an `Operations` object are
+    integrators, updaters, analyzers, tuners, and computes. An `Operations` can
+    only ever hold one integrator at a time. On the other hand, an `Operations`
+    object can hold any number of updaters, analyzers, tuners, or computes. To
+    see examples of these types of operations see `hoomd.hpmc.integrate`
+    or `hoomd.md.integrate` (integrators), `hoomd.update` (updaters) ,
+    `hoomd.tune` (tuners), `hoomd.dump` (analyzers), and `hoomd.md.thermo`
+    (computes).
+
+    A given instance of an operation class can only be added to a single
+    `Operations` container. Likewise, a single instance cannot be added to the
+    same `Operations` container more than once.
+
+    Note:
+        An `Operations` object is created by default when a new simulation is
+        created.
+    """
+
+    def __init__(self):
         self._compute = list()
         self._scheduled = False
-        self._updaters = SyncedList(OnlyType(_Updater),
+        self._updaters = SyncedList(OnlyType(Updater),
                                     _triggered_op_conversion)
-        self._analyzers = SyncedList(OnlyType(_Analyzer),
+        self._analyzers = SyncedList(OnlyType(Analyzer),
                                      _triggered_op_conversion)
-        self._tuners = SyncedList(OnlyType(_Tuner), lambda x: x._cpp_obj)
+        self._tuners = SyncedList(OnlyType(Tuner), lambda x: x._cpp_obj)
         self._integrator = None
 
         self._tuners.append(ParticleSorter())
 
-    def add(self, op):
+    def add(self, operation):
+        """Add operation to this container.
+
+        Adds the provide operation to the appropriate attribute of the
+        `Operations` instance.
+
+        Args:
+            operation (``operation``): A HOOMD-blue updater, analyzers, compute,
+                tuner, or integrator to add to the collection.
+
+        Raises:
+            RuntimeError: raised when an operation belonging to to this or
+                another `Operations` instance is passed.
+            TypeError: raised when the passed operation is not of a valid type.
+
+        Note:
+            Since only one integrator can be associated with an `Operations`
+            object at a time, this removes the current integrator when called
+            with an integrator operation. Also, the ``integrator`` property
+            cannot be set to ``None`` using this function. Use
+            ``operations.integrator = None`` explicitly for this.
+        """
         # calling _add is handled by the synced lists and integrator property.
         # we raise this error here to provide a more clear error message.
-        if op._added:
+        if operation._added:
             raise RuntimeError(
                 "Operation cannot be added to twice to operation lists.")
```

Comment:
The previous text doesn't make sense and could be more specific.
```suggestion
                "The provided operation has already been added to an Operations instance.")
```